app/models/user.rb

```diff
@@ -216,10 +220,127 @@ def send_devise_notification(notification, *args)
     devise_mailer.send(notification, self, *args).deliver_now_or_later
   end
 
-  def decorate
-    UserDecorator.new(self)
+  # Begin extracted methods from UserDecorator here
```

Comment:
now that UserDecorator is gone, I feel like this comment references something doesn't exist anymore. Maybe we could say "decoration methods" or something?

```suggestion
  #
  # Decoration methods
  #
```